packages/web/resolvers/user.ts

```diff
@@ -419,6 +426,48 @@ const UserMutations = extendType({
       },
     })
 
+    t.field('updateUserConfiguration', {
+      type: 'UserConfiguration',
+      args: {
+        digestEmailConfig: arg({
```

Comment:
@Lanny I also found that it was very straightforward here to do the same, simply using `arg()` and passing the string name of the enum for the `type` value. Simpler than we thought!